srcpkgs/sway/template

```diff
@@ -13,8 +13,9 @@ short_desc="Tiling Wayland compositor compatible with i3"
 maintainer="Olaf Mersmann <olafm@p-value.net>"
```

Comment:
this person last contributed in 2021, could you orphan this package? (feel free to adopt it if you are actively using it)

Reply:
I use Sway actively, so I'm fine with adopting it. @olafmersmann is this fine with you?

Reply:
Sure!